src/manifold/executor.clj

```diff
@@ -32,25 +32,42 @@
        (finally
          (.set executor-thread-local executor#)))))
 
+(defn- ^Thread new-thread
```

Comment:
Maybe add a docstring explaining why this fn exists?

Reply:
https://github.com/clj-commons/manifold/pull/210/commits/ab0ed826944b33028dabac5421e523e252bbe090

Reply:
Thanks, but that docstring feels like more of a "what" than a "why". Manifold itself never uses anything other than Thread, so this doesn't answer the newbie question "Why are you even using a fn for this at all?"

Maybe add a bit about "This fn exists to support Aleph using FastThreadLocalThread"

Reply:
Let's try with this?

```
  Creates a new `java.lang.Thread`.

  It represents the default implementation on `thread-factory` when the
  `new-thread-fn` argument is not passed.

  Some libraries, like Netty, behaves differently according the subclass of `java.lang.Thread`
  the code is running on. `io.netty.util.concurrent.FastThreadLocalThread` is the canonical example
  which provides performance advantage over `java.lang.Thread` to access Netty custom
  implementation of `ThreadLocal`.
```